dataclients/kubernetes/definitions/testdata/validation/route-with-invalid-backend-with-path.log

```diff
@@ -0,0 +1 @@
+backend address \"http://example.com/foo\" contains path\nbackend address \"http://example.com/foo/bar\" contains path\nbackend address \"/foo\" contains path\nbackend address \"/foo/bar\" contains path
```

Comment:
Tests fails because regexp doesn't work well with escape character "\" 

example https://go.dev/play/p/mbhk8lJI8xa

Reply:
After some tests made it work https://go.dev/play/p/yZUl2NCiP_n.